client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -72,6 +72,22 @@ export const getTask = (
 ) => {
 	let taskData = {};
 	switch ( task.id ) {
+		case 'site_created':
+			taskData = {
+				timing: 1,
+				title: translate( 'Site created' ),
+				description: translate(
+					"Next, we'll guide you through setting up and launching your site."
+				),
+				actionText: 'Get started',
+				...( ! task.isCompleted && {
+					actionDispatch: requestSiteChecklistTaskUpdate,
+					actionDispatchArgs: [ siteId, task.id ],
+				} ),
```

Comment:
This is probably redundant and can be removed.

Reply:
Indeed, this task will always be completed and actions are not needed since you introduced `actionAdvanceToNext`

Reply:
However, if there was ever a situation where this was not automatically complete (API error...etc) then it would be good to retain the ability to complete the step manually. As this is added conditionally it should be ok to retain.